core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/AbstractExperiment.java

```diff
@@ -38,329 +39,345 @@
 
 import org.w3c.dom.Document;
 
-import com.eaglegenomics.simlims.core.SecurityProfile;
-import com.eaglegenomics.simlims.core.User;
-
 import uk.ac.bbsrc.tgac.miso.core.data.impl.PlatformImpl;
 import uk.ac.bbsrc.tgac.miso.core.data.type.KitType;
 import uk.ac.bbsrc.tgac.miso.core.security.SecurableByProfile;
 import uk.ac.bbsrc.tgac.miso.core.util.CoverageIgnore;
 
+import com.eaglegenomics.simlims.core.SecurityProfile;
+import com.eaglegenomics.simlims.core.User;
+
 /**
  * Skeleton implementation of an Experiment
- *
+ * 
  * @author Rob Davey
  * @since 0.0.2
  */
 public abstract class AbstractExperiment implements Experiment {
-   public static final Long UNSAVED_ID = 0L;
-
-   @OneToOne(cascade = CascadeType.ALL)
-   private SecurityProfile securityProfile;
-
-   @Transient
-   public Document submissionDocument;
-
-   @ManyToOne(targetEntity = AbstractStudy.class, cascade = CascadeType.ALL)
-   private Study study = null;
-
-   // defines a pool on which this experiment will operate. This contains one or more dilutions of a sample
-   private Pool pool;
-
-   // defines the parent run which processes this experiment
-   private Run run;
-
-   private String title;
-   private String name;
-   private String description;
-   private String alias;
-
-   @Id
-   @GeneratedValue(strategy = GenerationType.AUTO)
-   private long experimentId = AbstractExperiment.UNSAVED_ID;
-   private String accession;
-
-   @OneToOne(targetEntity = PlatformImpl.class, cascade = CascadeType.ALL)
-   private Platform platform;
-
-   private Collection<Kit> kits = new HashSet<Kit>();
-
-   @Override
-   public Study getStudy() {
-      return study;
-   }
-
-   @Override
-   public void setStudy(Study study) {
-      this.study = study;
-   }
-
-   @CoverageIgnore
```

Comment:
Apparently you can use cobertura `ignoreTrivial` in the pom to ignore straight getters and setters.
I have never gotten it to work properly with maven though so this seems like the best solution :white_check_mark: